### PR TITLE
Report, Verification screens

### PR DESCRIPTION
**Pull request by kvey**
Some of these may be duplicates of wip
Screen Shot 2020-04-04 at 3 54 00 AM
Screen Shot 2020-04-04 at 4 28 33 AM

### CODE REVIEW

File: src/components/CodeInput.js [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+  hiddenInput: {
+    opacity: "0.0"
+  }
+
+});
+
+export const CodeInput = ({ onChange, codeValue, style }) => {
+  let code = ["", "", "", ""];
+  if (codeValue) {
+    codeValue.split("").map((c, i) => {
```
Comment (luxx) [inline src/components/CodeInput.js]: think you can just do
```
 input.split("")
```
and avoid mapping

File: src/screens/ReportScreen.js [hunk 2]
```diff
@@ -1,24 +1,179 @@
...
+            thumbColor={THUMB_COLOR}
+            ios_backgroundColor={IOS_SWITCH_BG_COLOR}
+            onValueChange={(v) => setSymptoms({...symptoms, "pressureInChest": v})}
+            value={symptoms["pressureInChest"]}
+          />
+          <Text style={styles.label}>
+            Pressure in Chest
+          </Text>
+        </View>
+        <View style={styles.row}>
```
Comment (luxx) [inline src/screens/ReportScreen.js]: can DRY this up put these Views into functions or new components, for example

```
const symptomRow = ({symptoms, type, description}) => {
  return (
    <View style={styles.row}>
      <Switch
        trackColor={TRACK_COLOR}
        thumbColor={THUMB_COLOR}
        ios_backgroundColor={IOS_SWITCH_BG_COLOR}
        onValueChange={(v) => setSymptoms({...symptoms, type: v})}
        value={symptoms[type]}
      />
      <Text style={styles.label}>
        {description}
      </Text>
    </View>
  )
}
```

then you can invoke from the view
```
{ symptomRow({ symptoms, type: "pressureInChest", description: "Pressure in Chest"}) }
{ symptomRow({ symptoms, type: "difficultyBreathing", description: "Difficulty Breathing"}) }
```
or evein in a loop

**Review by luxx — APPROVED** [review]

File: src/components/CodeInput.js (hunk 1 above)
Comment (kvey) [reply src/components/CodeInput.js]: Really there should be validation here limiting it to 4 chars. I'll do that!

File: src/screens/ReportScreen.js (hunk 2 above)
Comment (kvey) [reply src/screens/ReportScreen.js]: That's fair!

**Comment by BranescuMihai** [comment]
@kvey just realised this PR includes a screen from [the story](https://github.com/UpstreamInno/guardian_client/issues/3) I worked on. I [opened a PR](https://github.com/UpstreamInno/guardian_client/pull/24) on this, could you please take a look? :D

**Comment by luxx** [comment]
OTP is already in. We can probably still use ReportScreen.js, but I can pick it up in different PR (or you can). Closing this for now.

**Pull request closed without merging**